pkg/render/logstorage.go

```diff
@@ -450,20 +451,6 @@ func (es elasticsearchComponent) elasticsearchExternalService() *corev1.Service
 	}
 }
 
-func (es elasticsearchComponent) kibanaExternalService() *corev1.Service {
-	return &corev1.Service{
-		TypeMeta: metav1.TypeMeta{Kind: "Service", APIVersion: "v1"},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      KibanaServiceName,
-			Namespace: KibanaNamespace,
-		},
-		Spec: corev1.ServiceSpec{
-			Type:         corev1.ServiceTypeExternalName,
-			ExternalName: fmt.Sprintf("%s.%s.svc.%s", GuardianServiceName, GuardianNamespace, es.clusterDomain),
-		},
-	}
-}
-
```

Comment:
So this function is dead code and this is just to clean it up now, right?

Reply:
Yeah exactly